### DISCUSSION TITLE
make  CentOS Linux release 7.9.2009

### DISCUSSION THREAD

**Discussion by fquintieri** [abraunegg/onedrive#1516] [Q&A] [answered]
Hi 

I have a problem during command make. My error is 
```
[root@tableau onedrive]# make -f Makefile.in clean; make -f Makefile.in
rm -f onedrive onedrive.o version
rm -rf autom4te.cache
rm -f config.log config.status
if [ -f .git/HEAD ] ; then \
        git describe --tags > version ; \
else \
        echo @PACKAGE_VERSION@ > version ; \
fi
make: DC@: Command not found
make: *** [onedrive] Error 127
[root@tableau onedrive]#
```
Does anyone have any idea?

thanks

**Comment by abraunegg** [comment, marked as answer]
@fquintieri 
This error is explained in detail here: https://github.com/abraunegg/onedrive/wiki/Frequently-Asked-Questions#make-command-during-installation-encounters-error-127

To resolve you need to follow the instructions on building the client as detailed here: https://github.com/abraunegg/onedrive/blob/master/docs/INSTALL.md#compilation--installation

1. Install the platfrom dependancies for your Linux OS
2. Activate your DMD or LDC compiler
3. Clone the GitHub repository, run configure and make, then install
4. Deactivate your DMD or LDC compiler

**Reply by fquintieri** [reply]
Hi, 

thanks for your answer, but I had already read about this erro in   https://github.com/abraunegg/onedrive/wiki/Frequently-Asked-Questions#make-command-during-installation-encounters-error-127.

I did exactly what was written, but it is still not work. Could you see what happened with print screen in the attached file?

![print_screen_err](https://user-images.githubusercontent.com/73612581/122480314-fced6500-cfa2-11eb-96f3-47e55d16a524.png)

**Reply by abraunegg** [reply]
@fquintieri 
You are simply just not following the directions.

When you install DMD or LDC it *gives* you the full activate command to enable the compiler - this is the command you use in point 2 above.

Suggest that you remove your 'dlang' and 'onedrive' directories and start again .. follow the steps, step-by-step - reading the output as provided, especially when you install the compiler.

**Comment by fquintieri** [comment]
thank you so much. I got it
